### PR TITLE
Update lwa4p_extended.urdf.xacro

### PR DESCRIPTION
**Pull request by thiagodefreitas**
Proposing this as according to the tests on the Schunk LWA4D, the limits should be reduced from the HW limit in a range of 0.01 radians

### CODE REVIEW

**Comment by thiagodefreitas** [comment]
@ipa-fmw, @ipa-fxm, another approach would be to use the soft limits

**Comment by floweisshardt** [comment]
@thiagodefreitas is this really enough if you run fullspeed into the limits?

**Comment by thiagodefreitas** [comment]
For the lwa4d, I could achieve the position with full speed without reaching the HW limit. I am going to test it later on the lwa4p_extended to see if it has the same behavior.

**Comment by floweisshardt** [comment]
please merge with ipa320 first

**Comment by thiagodefreitas** [comment]
@ipa-fmw : Done

File: schunk_description/urdf/lwa4p_extended/lwa4p_extended_without_base.urdf.xacro [hunk 1]
```diff
@@ -15,7 +15,7 @@
       <axis xyz="0 0 1"/>
       <calibration rising="${arm_1_ref}"/>
       <dynamics damping="0" friction="0"/>
-      <limit effort="81.5" velocity="0.42" lower="-4.712" upper="4.712"/>   <!--Theoricarl limit +/-270 , Real Limit +/- 270--> 
+      <limit effort="81.5" velocity="0.42" lower="-4.702" upper="4.702"/>   <!--Theoricarl limit +/-270 , Real Limit +/- 270-->
```
Comment (fmessmer) [inline schunk_description/urdf/lwa4p_extended/lwa4p_extended_without_base.urdf.xacro]: Theoricarl -> Theoretical

I think it should be documented that we substracted 0.01 - either as a comment or by writng `${-4.712 + 0.01}` ?

What's the difference between theoretical and real limits? Are the theoretical limits what the motors (Schunk modules) are capable of and real what is possible in the lwa4p_extended configuration?

Regarding usage of softlimits:
What were the results of the test using the ros_control joint_limits_interfaces? Where they considered correctly there if the the safety_controller tag is activated? Any other observations with the safety_controller tag?

Comment (thiagodefreitas) [inline schunk_description/urdf/lwa4p_extended/lwa4p_extended_without_base.urdf.xacro]: @ipa-fxm: Concerning the difference between theoretical and real, your assumption is correct.

Comment (floweisshardt) [inline schunk_description/urdf/lwa4p_extended/lwa4p_extended_without_base.urdf.xacro]: +1 for `${-4.712 + 0.01}`

even better: definde a property `safety_offest=0.01` and use `${-4.712 + safety_offset}`

@thiagodefreitas: please adapt

**Comment by thiagodefreitas** [comment]
Added the safety_offset property. After https://github.com/ipa320/schunk_modular_robotics/pull/124 and https://github.com/ipa320/schunk_modular_robotics/pull/120

**Comment by floweisshardt** [comment]
visual review: OK

**Comment by thiagodefreitas** [comment]
safety_offset also for lwa4d and lwa4p.

**Comment by floweisshardt** [comment]
visual review: OK

**Pull request merged**